views/fe-utils/graphTableRowDataUtil.js

```diff
@@ -0,0 +1,15 @@
+import { TOTAL_LABEL } from '../../constants'
+import { studyCountsToPercentage } from './helpers'
+
+export const graphTableRowData = (site) => {
```

Comment:
this is not a site but graph data by site, right?

Reply:
this is a very specific helper function (it generates site-specific table row data for a bar chart) with a generic name in a generic folder. I think we want this to be more specifically named or more generally named but placed in a specific location, like in the `GraphTable` directory.